flink/test_flink.py

```diff
@@ -29,6 +30,10 @@ def __run_test_file(self, name, yarn_session):
       ("HA", ["m-0", "m-1", "m-2"]),
   )
   def test_flink(self, configuration, machine_suffixes):
+    # Skip on 2.0+ version of Dataproc because it's not supported
+    if self.getImageVersion() >= pkg_resources.parse_version("2.0"):
+      return
```

Comment:
Is there a way to mark the test as skipped?

Reply:
Done.